lib/graphql/schema/field.rb

```diff
@@ -34,7 +34,8 @@ class Field
       # @param field [GraphQL::Field] **deprecated** for compatibility with <1.8.0
       # @param function [GraphQL::Function] **deprecated** for compatibility with <1.8.0
       # @param camelize [Boolean] If true, the field name will be camelized when building the schema
-      def initialize(name, return_type_expr = nil, desc = nil, null: nil, field: nil, function: nil, description: nil, deprecation_reason: nil, method: nil, connection: nil, max_page_size: nil, resolve: nil, introspection: false, hash_key: nil, camelize: true, extras: [], &definition_block)
+      # @param complexity [Numeric] When provided, set the complexity for this field. Default 1.
```

Comment:
(for the record, YARD will pick up default values from the source code, no need to document them here in my opinion, for example: http://www.rubydoc.info/gems/graphql/GraphQL%2FObjectType:implements)

Reply:
YARD so smart! I didn't realise this, thanks!